modules/c/pages/gs-install.adoc

```diff
@@ -335,6 +426,57 @@ In this instance the release comprises a _ready to adapt_ application project.
 
 Once the install is finished, you can build and run this skeleton app.
 
+=== Install Vector Search for Android
+
+To use Vector Search in your Android applications, follow the steps below:
+
+. Download and extract the link:https://www.couchbase.com/downloads/?family=couchbase-lite#LiteExtensions[Vector Search extension] into your project location.
+. The package must be installed to the location in your project where the library can be accessed and loaded while the executable is running.
+.. The Vector Search download for CBL-C only contains the Vector Search libraries needed to include in your app.
+.. Steps to include the prebuilt native library can be found link:https://developer.android.com/studio/projects/gradle-external-native-builds[here].
+. Create a Android app project in Android Studio
+. The location of the native library can be found using the following snippet:
+// FOR 3.2.0 GA >> . For `x86_64` architectures: `com.couchbase.lite:couchbase-lite-android-vector-search-x86_64-{vs-version-maintenance}`
+
+
+[{tabs}]
+=====
+
+Default Packaging:: 
++
+--
+[source, java]
+----
+String getExtensionPath(Context context) {
+  String packagePath = context.getPackageResourcePath();
+  if (packagePath == null) { return null; }
+
+  String arch = getArch();
+  if (arch == null) { return null; }
+
+  return packagePath + "!/lib/" + arch; // "!" is important for locating non-extracted library.
+}
+
+String getArch() {
+  final List<String> abis = Arrays.asList(Build.SUPPORTED_ABIS);
+  if (abis.contains()) { return "arm64-v8a"; }
+  if (abis.contains("x86_64")) { return "x86_64"; }
+  return null;
+}
+----
+--
+Legacy Packaging::
++
+--
+[source, java]
+----
+static String getExtensionPath() {
+  return context.getApplicationInfo().nativeLibraryDir
+}
+----
+
+--
+=====
```

Comment:
Same as the other platforms, users will need to set the extension path as well.

Reply:
Done!